src/utils/fastmath.h

```diff
@@ -77,9 +77,4 @@ inline float FastLog(const float a) {
 // Fast approximate exp(x). Does only limited range checking.
 inline float FastExp(const float a) { return FastPow2(1.442695040f * a); }
 
-// Fast logit for more readable code.
-inline float FastLogit(const float a) {
```

Comment:
The include is unnecessary now https://github.com/LeelaChessZero/lc0/blob/770ca152e168b0f8d28a4e8e7eb7ef8bfcb546b9/src/mcts/node.h#L42